triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Converter.scala

```diff
@@ -74,14 +74,7 @@ object Converter {
 
   private case class AnyContractId(templateId: Identifier, contractId: ContractId)
   private case class AnyTemplate(ty: Identifier, arg: SValue)
-  sealed abstract class AnyChoice extends Product with Serializable {
-    def name: ChoiceName
-    def arg: SValue
-  }
-  object AnyChoice {
-    final case class Template(name: ChoiceName, arg: SValue) extends AnyChoice
-    final case class Interface(ifaceId: Identifier, name: ChoiceName, arg: SValue) extends AnyChoice
-  }
+  private case class AnyChoice(name: ChoiceName, arg: SValue)
```

Comment:
unrelated, do we have a good reason for duplicating this (and others) between the script and trigger `Converter`s?